packages/core/README.md

```
@@ -108,27 +108,55 @@ function handleReceipt(receipt: Receipt) {
+
+~~~~ typescript
+import { createFailedReceipt } from "@upyo/core";
+
+const receipt = createFailedReceipt("HTTP 429: Too Many Requests", {
+  provider: "example",
+  statusCode: 429,
+  retryAfterMilliseconds: 30_000,
+});
+~~~~
+
 ### Implementing custom transports
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)